lib/toShahmukhi.js

```diff
@@ -38,7 +38,7 @@ const shahmukhiMappings = {
   'ੑ': 'ھ',
   'ੱ': 'ّ',
   '੍ਵ': 'و',
-  'ਃ': ': ',
+  'ਃ': ':',
```

Comment:
Try to avoid changing stuff outside of the scope of the PR in future

Reply:
Oh oops, was supposed to commit that to the other PR.